### ISSUE TITLE
`Cannot find module 'esm-env'`

### ISSUE THREAD

**Issue by ogrotten** [efstajas/svelte-stepper#3]
```
Cannot find module 'esm-env' imported from '/projectpath/node_modules/@efstajas/svelte-stepper/package/stepper.svelte'
```

Is this a complaint about too old a version of svelte/kit or lack of TS?

**Comment by efstajas** [comment]
Hey @ogrotten! I've just published v0.1.5, which explicitly depends on `esm-env`. Could you try again with that?

**Comment by ogrotten** [comment]
Thanks for the fix, it's working great now!
